app/lib/actions/foreman_ansible/helpers/host_common.rb

```diff
@@ -29,10 +29,6 @@ def fill_continuous_output(continuous_output)
         rescue StandardError => e
           continuous_output.add_exception(_('Error loading data from proxy'), e)
         end
-
-        def find_options
```

Comment:
The entire module seems like a leftover from when foreman ansible implemented its own actions. A quick grep didn't find it being used anywhere. How about I drop it entirely?